feature/tour page main section fetch & render data

Ahmed-Fahmy-Dev opened the pull request:
- tour name rendered as h1
- about tour rendered with content
- reasons to choose rendered with content

Ahmed-Fahmy-Dev committed:
update: main section rendered after fetch:
- tour name rendered as h1
- about tour rendered with content
- reasons to choose rendered with content

HIMDanny commented on client/src/pages/TourPage/TourPage.js:
```diff
@@ -147,19 +147,21 @@ const TourPage = () => {
 
   return (
     <>
+      <div>
+        <ImageGallery />
+      </div>
```
Suggest deleting it cause `ImageGallery` already placed in `HeaderContent > Container`

Ahmed-Fahmy-Dev replied:
done

HIMDanny commented on client/src/pages/TourPage/TourPage.js:
```diff
@@ -147,19 +147,21 @@ const TourPage = () => {
+      </div>
       <HeaderContent>
         <Container>
           <Typography align="center" variant="h1" mt={17} mb={5} fontSize="50px">
```
Suggest capitalizing first letter of the title

```suggestion
          <Typography
            align="center"
            variant="h1"
            mt={17}
            mb={5}
            fontSize="50px"
            sx={{
              ':first-letter': {
                textTransform: 'capitalize',
              },
            }}
          >
```

Ahmed-Fahmy-Dev replied:
done

Ahmed-Fahmy-Dev committed:
fix: styling and HTML
- capitalize first letter of h1
- removed unneeded ImageGallery component from JSX

Ahmed-Fahmy-Dev committed:
Merge branch 'develop' into feature/tourPage-mainSection-fetchData

AltysS commented:
Great job. Nothing to add here

Ahmed-Fahmy-Dev committed:
Merge branch 'develop' into feature/tourPage-mainSection-fetchData

Ahmed-Fahmy-Dev committed:
Merge branch 'develop' into feature/tourPage-mainSection-fetchData
